extensions/PhabBugz/lib/Revision.pm

```diff
@@ -52,6 +52,7 @@ has stack_graph_raw  => (
 );
 has subscriber_count => (is => 'ro',   isa => Int);
 has bug              => (is => 'lazy', isa => Object);
+has uplift_request   => (is => 'ro',   isa => Dict [ slurpy Any ]);
```

Comment:
Is this how I would add the `uplift_request` field without having to specify the exact values of the keys?

Reply:
Yeah